src/javascripts/ng-admin/Crud/list/ListController.js

```diff
@@ -1,4 +1,5 @@
 /*global define*/
+import Entry from 'admin-config/lib/Entry';
 
 define(function () {
```

Comment:
Not specific to this PR, but can you remove the `define`, as it is just requirejs obsolete code? Just `export default` the controller function.